tslint.json

```diff
@@ -83,8 +83,7 @@
       }
     },
     "typedef": [
-      true,
-      "call-signature"
```

Comment:
Hierdie was nog 'n silly strict setting. Net "call-signature" verwyder. Ons kan nog in die toekoms terug kom hiernatoe. Vir nou dink ek kan dit meer confusion veroorsaak as wat nodig.

Basically, dit force jou methods om return types te wys